OWNERS_ALIASES

```diff
@@ -0,0 +1,11 @@
+ 
+# See the CONTRIBUTING docs: https://github.com/openshift/origin/blob/master/CONTRIBUTING.adoc
+
+aliases:
+  shiftstack-team:
+    - Fedosin
```

Comment:
Nit: could you sort alphabetically?

Reply:
Sure! The original order was taken from https://github.com/openshift/cluster-api-provider-openstack/blob/master/OWNERS_ALIASES

Reply:
Done

Reply:
> The original order was taken from

ih ih that's me https://github.com/openshift/cluster-api-provider-openstack/commit/ef2821d45f0652f1c34be44511f45de57cc7bad6#diff-0de52d14f35759dfa67139f90c82bd7f